lib/DDG/Spice/Tides.pm

```diff
@@ -8,6 +8,7 @@ spice is_cached => 1; #we can cache the response and just filter out the tide ev
 spice to => 'http://api.wunderground.com/api/{{ENV{DDG_SPICE_WUNDERGROUND_APIKEY}}}/tide/q/$1.json?callback={{callback}}';
 
 triggers any => "tide", "tides";
+triggers start => "tides for", "tide for", "tide in", "tides in", "tide times in";
```

Comment:
Could you add some test cases for `tide for ` and `tides in` too? Thanks :smile: